src/lib/app_registry.tsx

```diff
@@ -91,7 +91,16 @@ class WorksForYou extends RootContainer {
   }
 }
 
+class MyProfile extends RootContainer {
```

Comment:
Not really anything special in the naming, just wanted a name so I could move on. `MyAccount` is probably a better one IMO. This can be a blocker on the PR.